src/test/java/org/tarantool/schema/ClientSchemaIT.java

```diff
@@ -0,0 +1,226 @@
+package org.tarantool.schema;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.tarantool.TestUtils.makeDefaultClientConfig;
+
+import org.tarantool.ServerVersion;
+import org.tarantool.TarantoolClientConfig;
+import org.tarantool.TarantoolClientImpl;
+import org.tarantool.TarantoolTestHelper;
+import org.tarantool.TestAssumptions;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+@DisplayName("A schema meta")
+public class ClientSchemaIT {
+
+    private static TarantoolTestHelper testHelper;
+
+    private TarantoolClientImpl client;
+
+    @BeforeAll
+    public static void setupEnv() {
+        testHelper = new TarantoolTestHelper("client-schema-it");
+        testHelper.createInstance();
+        testHelper.startInstance();
+    }
+
+    @AfterAll
+    public static void teardownEnv() {
+        testHelper.stopInstance();
+    }
+
+    @BeforeEach
+    public void setup() {
+        TarantoolClientConfig config = makeDefaultClientConfig();
+
+        client = new TarantoolClientImpl(
+            TarantoolTestHelper.HOST + ":" + TarantoolTestHelper.PORT,
+            config
+        );
+    }
+
+    @AfterEach
+    public void tearDown() {
+        client.close();
+        testHelper.executeLua("box.space.count_space and box.space.count_space:drop()");
+    }
+
+    @Test
+    @DisplayName("fetched a space with its index")
+    void testFetchSpaces() {
+        testHelper.executeLua(
+            "box.schema.space.create('count_space', { format = " +
+                "{ {name = 'id', type = 'integer'}," +
+                "  {name = 'counts', type = 'integer'} }" +
+                "})"
+        );
+        testHelper.executeLua("box.space.count_space:create_index('pk', { type = 'TREE', parts = {'id'} } )");
+
+        TarantoolSchemaMeta meta = new TarantoolSchemaMeta(client);
```

Comment:
Is it planned as public API? I think it should be documented at least briefly if so.

Reply:
I didn't have the intention to do client background services being public. Should they?

Reply:
I would say 'no', they should not be public until at least we'll see a demand. Should the class be declared as non-public in the case?

Reply:
`TarantoolSchemaMeta` belongs to its own package `org.tarantool.schema`. This means we need to make the class public because of being visible in TarantoolClient. In this case, a class usually is marked as internal in java-doc (for instance, "Internal class, it is not a part of public API." sentence at the end of the description).